packages/api/src/trpc.ts

```diff
@@ -120,6 +120,16 @@ const isAuthed = t.middleware(async ({ next, ctx }) => {
     throw new TRPCError({ code: "UNAUTHORIZED" });
   }
 
+  await ctx.db.participant.upsert({
+    where: {
+      id: 1,
+    },
+    update: {},
+    create: {
+      name: "Test",
```

Comment:
I don't think this is needed here, since to my understanding a `Participant` presents a child that the `User` is managing (see my comment below).  

In the case of the `upsert` for users we're inserting the user into our database if they aren't there yet. However for participants we don't have to enforce that since maybe the user just signed up and hasn't added a participant in the database yet to link to their account.

Reply:
This is just a stopgap fix right now, as every user needs a participant. Thus if a user isn't created in a db, then we also need to have a partipicant assigned to them. When we add the admin functionality of adding whitelisted users, we can remove this logic.

Reply:
What's our plan for adding participants to a user? E.g. user will go to a dashboard and add their participant? But what if they want to select from existing kids because another user has already created the participant? Will we give them a dropdown list to pick from?

Reply:
I think it will be a part of the whitelisting process that the admin will perform -- so they will create a participant and the necessary users